### DISCUSSION TITLE
Error after pacakage and framework update

### DISCUSSION THREAD

**Discussion by vinnaykulkarni** [serenity-is/Serenity#6715] [Help]
I have updated the project from .Net core 5 to .Net Core 7 and accordingly, packages got updated, my project gets compiled properly however when I try to add or update existing data I am getting the following error.

**Method not found: 'Void Serenity.Data.Row`1.set_TrackAssignments(Boolean)'.**

**Comment by reach2rv** [comment]
> I have updated the project from .Net core 5 to .Net Core 7 and accordingly, packages got updated, my project gets compiled properly however when I try to add or update existing data I am getting the following error.
> 
> **Method not found: 'Void Serenity.Data.Row`1.set_TrackAssignments(Boolean)'.**

@vinaykulkarni89 - please check your hotmail account, let me know if we can connect over it. Thank you!

**Comment by LSNicholls** [comment]
Is it allowed to update from .Net Core 6 to .Net Core 7?  I thought Serenity was dependent on 6 and we weren't supposed to do that.

Additionally given that 8 is the next LTS version maybe that would be the next supported version for Serenity?

**Reply by VictorTomaili** [reply]
The .net 7 is your responsibility. We didn't offer .net 7 support, but theoretically, there is no restriction to updating it. We prefer to continue with the LTS releases.

**Comment by LSNicholls** [comment]
@VictorTomaili  thank you for confirming policy.
